client/html/pages/signup/students.html

```diff
@@ -60,7 +60,7 @@
                   </div>
                   <div class="col-sm-3">
                     <div class="form-group">
-                      <input type="text" class="form-control" id="nascimento" name="nascimento" required>
+                      <input type="text" class="form-control" id="nascimento" name="nascimento" required onfocus="this.blur()">
```

Comment:
I had an idea of how to improve this, I will send a PR to you with what I thought, then you tell me if you liked it ;)